policy/modules/system/fstools.if

```diff
@@ -321,3 +321,32 @@ interface(`fstools_manage_swap_files',`
 	allow $1 swapfile_t:file manage_file_perms;
 ')
 
+########################################
+## <summary>
+##	Manage swapfile.
```

Comment:
Please change this description. Something like "Create objects in the runtime directory with an automatic type transition to the fsadm runtime type."

Reply:
Forgot about this, sry.